### PR TITLE
closes #1923.  fix broken isometric example

### LINKED ISSUE #1923: $ go run examples/isometric/main.go # command-line-arguments examples/isometric/main.go:31:12: undefined: NewGame
`examples/isometric` does not run in it's current state.

```
$ go run examples/isometric/main.go
# command-line-arguments
examples/isometric/main.go:31:12: undefined: NewGame
```

### PR DESCRIPTION
**Pull request by jacobsalmela**
the Spritesheet_png was not included with go generate.  I also moved the
non-main files into their own folder to better match other examples in the repo.

Signed-off-by: Jacob Salmela <me@jacobsalmela.com>

### CODE REVIEW

**Comment by hajimehoshi** [comment]
> the Spritesheet_png was not included with go generate.

Good point so we should fix only examples/resources/generate.go We don't have to touch other files.

**Comment by jacobsalmela** [comment]
Wilco.  I'll adjust that.

File: examples/resources/generate.go [hunk 1]
```diff
@@ -24,6 +24,7 @@
 //go:generate file2byteslice -package=images -input=./images/smoke.png -output=./images/smoke.go -var=Smoke_png
 //go:generate file2byteslice -package=images -input=./images/tile.png -output=./images/tile.go -var=Tile_png
```
Comment (hajimehoshi) [inline examples/resources/generate.go:25]: Optional: you can move this so that the list is in alphabetical order

**Review by hajimehoshi — APPROVED** [review]
LGTM, thank you very much!

File: examples/resources/generate.go (hunk 1 above)
Comment (jacobsalmela) [reply examples/resources/generate.go:25]: It looks like there are other entries already not in alphabetical order... 🤷

Comment (hajimehoshi) [reply examples/resources/generate.go:25]: Ack

**Pull request merged**